eventmesh-common/src/main/java/org/apache/eventmesh/common/EventMeshCommon.java

```diff
@@ -29,16 +29,5 @@ public class EventMeshCommon {
      */
     public static final int DEFAULT_TIME_OUT_MILLS = 20 * 1000;
 
-    /**
-     * USERAGENT for PUB
-     */
-    public static final String USER_AGENT_PURPOSE_PUB = "pub";
-
-    /**
-     * USERAGENT for SUB
-     */
-    public static final String USER_AGENT_PURPOSE_SUB = "sub";
-
     public static final String EM_MESSAGE_PROTOCOL_NAME = "eventmeshmessage";
-    public static final String OPEN_MESSAGE_PROTOCOL_NAME = "openmessage";
 }
```

Comment:
`OPEN_MESSAGE_PROTOCOL_NAME` shouldn't be removed from `EventMeshCommon` because it may be used by SDK clients. ref: https://eventmesh.apache.org/zh/docs/sdk-java/http